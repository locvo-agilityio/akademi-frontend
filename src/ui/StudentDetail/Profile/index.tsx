import { Avatar, Box, Flex, Image, Text, VStack } from '@chakra-ui/react';

// Images
import BgStudent from '@/images/bg-student.webp';

// Components
import Contact from '../Contact';

interface IProfileProps {
  email?: string;
  phone?: string;
  address?: string;
  parentName?: string;
  avatar?: string;
}

const Profile = ({
  email = '',
  phone = '',
  address = '',
  parentName = '',
  avatar = '',
}: IProfileProps) => (
  <VStack
    w="full"
    flexDirection="column"
    minH={498}
    borderRadius="2xl"
    bgColor="white"
    justifyContent="space-between"
  >
    <Box
      w="full"
      h={200}
      position="relative"
      bgColor="primary"
      borderTopRadius="2xl"
    >
      <Image
        w="full"
        h={200}
        objectFit="cover"
        objectPosition="right"
        src={BgStudent}
        alt="Background Student"
      />

      <Box position="absolute" top="50%" left={8} w="full">
        <Flex
          w={160}
          h={160}
          borderRadius="full"
          bgColor="white"
          align="center"
          justify="center"
        >
          <Avatar w="full" h="full" p={2} src={avatar} />
        </Flex>

        <Text as="h2" size="lg" mt={6} fontWeight="bold" color="darkBlue">
          {parentName}
        </Text>
        <Text as="p" mt={2} color="whiteSmoke">
          Student
        </Text>
      </Box>
    </Box>

    <Contact
      email={email}
      parentName={parentName}
      address={address}
      phone={phone}
    />
  </VStack>
);

export default Profile;
