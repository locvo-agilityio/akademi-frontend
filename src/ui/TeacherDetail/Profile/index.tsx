'use client';

import { memo } from 'react';
import { Avatar, Box, Flex, Image, Text, VStack } from '@chakra-ui/react';

// Images
import BgTeacher from '@/images/bg-teacher.webp';

// Components
import Contact from '../Contact';

// Components
import Information from '../Information';

interface IProfileTeacherProps {
  name?: string;
  email?: string;
  phone?: string;
  subject?: string;
  avatar?: string;
  address?: string;
  description?: string;
  degree?: string;
  university?: string;
  expertise?: string;
  startEducation?: string;
  endEducation?: string;
}

const ProfileTeacher = ({
  name = '',
  email = '',
  phone = '',
  subject = '',
  avatar = '',
  address = '',
  description = '',
  degree = '',
  university = '',
  expertise = '',
  startEducation = '',
  endEducation = '',
}: IProfileTeacherProps) => (
  <VStack
    w="full"
    flexDirection="column"
    borderRadius="2xl"
    bgColor="white"
    justifyContent="space-between"
    alignItems="flex-start"
  >
    <Box
      w="full"
      h={200}
      position="relative"
      bgColor="primary"
      borderTopRadius="2xl"
    >
      <Image
        w="full"
        h={200}
        objectFit="cover"
        objectPosition="right"
        src={BgTeacher}
        alt="Background Student"
      />

      <Box position="absolute" maxW={200} top="50%" left={8} w="full">
        <Flex
          w={160}
          h={160}
          borderRadius="full"
          bgColor="white"
          align="center"
          justify="center"
        >
          <Avatar w="full" h="full" p={2} src={avatar} />
        </Flex>

        <Text as="h2" size="lg" mt={6} fontWeight="bold" color="darkBlue">
          {name}
        </Text>
        <Text as="p" mt={2} color="whiteSmoke" fontWeight="semibold">
          {subject} Teacher
        </Text>
      </Box>
    </Box>

    <Flex mt={160} p={8} w="full" flexDirection="column">
      <Contact email={email} address={address} phone={phone} />

      <Information
        degree={degree}
        university={university}
        expertise={expertise}
        description={description}
        startEducation={startEducation}
        endEducation={endEducation}
      />
    </Flex>
  </VStack>
);

export default memo(ProfileTeacher);
