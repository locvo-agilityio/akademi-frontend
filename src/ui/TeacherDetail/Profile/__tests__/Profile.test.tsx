import { render } from '@testing-library/react';

import Profile from '..';

describe('Profile Teacher', () => {
  it('should render default value successfully', () => {
    const { container } = render(<Profile />);

    expect(container).toMatchSnapshot();
  });

  it('should render value successfully', () => {
    const { container } = render(
      <Profile
        address="Jakarta"
        email="john.doe@example.com"
        phone="08123456789"
        subject="Math"
        avatar="https://example.com/avatar.jpg"
        name="John Doe"
        degree="Bachelor of Science"
        university="University of Example"
        expertise="Mathematics"
        startEducation="2015"
        endEducation="2020"
      />,
    );

    expect(container).toMatchSnapshot();
  });
});
