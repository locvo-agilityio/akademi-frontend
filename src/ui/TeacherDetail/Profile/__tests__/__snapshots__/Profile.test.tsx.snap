// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Teacher should render default value successfully 1`] = `
<div>
  <div
    class="chakra-stack css-8qeac1"
  >
    <div
      class="css-9npxv6"
    >
      <img
        alt="Background Student"
        class="chakra-image css-1m8o377"
        src="test-file-stub"
      />
      <div
        class="css-tyv5ff"
      >
        <div
          class="css-1367w0v"
        >
          <span
            class="chakra-avatar css-131besb"
          >
            <svg
              aria-label=" avatar"
              class="chakra-avatar__svg css-16ite8i"
              role="img"
              viewBox="0 0 128 128"
            >
              <path
                d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                fill="currentColor"
              />
              <path
                d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                fill="currentColor"
              />
            </svg>
          </span>
        </div>
        <h2
          class="chakra-text css-hn8sh4"
        />
        <p
          class="chakra-text css-7j6s3t"
        >
           Teacher
        </p>
      </div>
    </div>
    <div
      class="css-1j1ywq9"
    >
      <div
        class="chakra-stack css-10hx9m5"
      >
        <div
          class="css-r2hvuq"
        >
          <div
            class="css-fnjvi5"
          >
            <svg
              class="chakra-icon css-16n3ul6"
              fill="none"
              focusable="false"
              height="20"
              viewBox="0 0 16 20"
              width="16"
            >
              <path
                d="M8 0C5.87827 0 3.84344 0.842855 2.34315 2.34315C0.842855 3.84344 0 5.87827 0 8C0 13.4 7.05 19.5 7.35 19.76C7.53113 19.9149 7.76165 20.0001 8 20.0001C8.23835 20.0001 8.46887 19.9149 8.65 19.76C9 19.5 16 13.4 16 8C16 5.87827 15.1571 3.84344 13.6569 2.34315C12.1566 0.842855 10.1217 0 8 0ZM8 17.65C5.87 15.65 2 11.34 2 8C2 6.4087 2.63214 4.88258 3.75736 3.75736C4.88258 2.63214 6.4087 2 8 2C9.5913 2 11.1174 2.63214 12.2426 3.75736C13.3679 4.88258 14 6.4087 14 8C14 11.34 10.13 15.66 8 17.65ZM8 4C7.20887 4 6.43552 4.2346 5.77772 4.67412C5.11992 5.11365 4.60723 5.73836 4.30448 6.46927C4.00173 7.20017 3.92252 8.00444 4.07686 8.78036C4.2312 9.55628 4.61216 10.269 5.17157 10.8284C5.73098 11.3878 6.44371 11.7688 7.21964 11.9231C7.99556 12.0775 8.79983 11.9983 9.53073 11.6955C10.2616 11.3928 10.8864 10.8801 11.3259 10.2223C11.7654 9.56448 12 8.79113 12 8C12 6.93913 11.5786 5.92172 10.8284 5.17157C10.0783 4.42143 9.06087 4 8 4ZM8 10C7.60444 10 7.21776 9.8827 6.88886 9.66294C6.55996 9.44318 6.30362 9.13082 6.15224 8.76537C6.00087 8.39991 5.96126 7.99778 6.03843 7.60982C6.1156 7.22186 6.30608 6.86549 6.58579 6.58579C6.86549 6.30608 7.22186 6.1156 7.60982 6.03843C7.99778 5.96126 8.39991 6.00087 8.76537 6.15224C9.13082 6.30362 9.44318 6.55996 9.66294 6.88886C9.8827 7.21776 10 7.60444 10 8C10 8.53043 9.78929 9.03914 9.41421 9.41421C9.03914 9.78929 8.53043 10 8 10Z"
                fill="white"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1v07gju"
          />
        </div>
        <div
          class="css-r2hvuq"
        >
          <div
            class="css-fnjvi5"
          >
            <svg
              class="chakra-icon css-16n3ul6"
              fill="none"
              focusable="false"
              height="20"
              viewBox="0 0 21 20"
              width="21"
            >
              <path
                d="M18.4401 11.0001C18.2201 11.0001 17.9901 10.9301 17.7701 10.8801C17.3246 10.7819 16.8868 10.6516 16.4601 10.4901C15.9962 10.3213 15.4862 10.3301 15.0284 10.5147C14.5706 10.6993 14.1972 11.0467 13.9801 11.4901L13.7601 11.9401C12.7861 11.3983 11.8911 10.7253 11.1001 9.94011C10.3149 9.14913 9.64192 8.25411 9.1001 7.28011L9.5201 7.00011C9.96347 6.78303 10.3109 6.40964 10.4955 5.9518C10.6801 5.49396 10.6889 4.98402 10.5201 4.52011C10.3613 4.09253 10.231 3.65491 10.1301 3.21011C10.0801 2.99011 10.0401 2.76011 10.0101 2.53011C9.88867 1.82573 9.51973 1.18785 8.96972 0.731352C8.41972 0.274849 7.7248 0.0297211 7.0101 0.0401122H4.0101C3.57913 0.0360657 3.15235 0.124926 2.75881 0.300643C2.36527 0.47636 2.01421 0.734808 1.72953 1.05839C1.44485 1.38198 1.23324 1.76311 1.10909 2.17583C0.984942 2.58855 0.95118 3.02317 1.0101 3.45011C1.54284 7.63949 3.45613 11.532 6.44775 14.5127C9.43938 17.4935 13.3388 19.3926 17.5301 19.9101H17.9101C18.6475 19.9112 19.3595 19.6406 19.9101 19.1501C20.2265 18.8672 20.4792 18.5203 20.6516 18.1324C20.8239 17.7446 20.9121 17.3245 20.9101 16.9001V13.9001C20.8979 13.2055 20.6449 12.5367 20.1944 12.0078C19.744 11.4789 19.1239 11.1227 18.4401 11.0001ZM18.9401 17.0001C18.9399 17.1421 18.9095 17.2824 18.8509 17.4117C18.7923 17.5411 18.7068 17.6564 18.6001 17.7501C18.4884 17.8466 18.3577 17.9187 18.2165 17.9617C18.0753 18.0048 17.9267 18.0178 17.7801 18.0001C14.035 17.5199 10.5563 15.8066 7.89282 13.1304C5.2293 10.4542 3.53251 6.96745 3.0701 3.22011C3.05419 3.07363 3.06813 2.92544 3.1111 2.7845C3.15407 2.64356 3.22517 2.5128 3.3201 2.40011C3.41381 2.29344 3.52916 2.20795 3.65848 2.14933C3.7878 2.09071 3.92812 2.06029 4.0701 2.06011H7.0701C7.30265 2.05494 7.52972 2.13099 7.71224 2.27518C7.89476 2.41937 8.02131 2.62268 8.0701 2.85011C8.1101 3.12345 8.1601 3.39345 8.2201 3.66011C8.33562 4.18726 8.48936 4.70529 8.6801 5.21011L7.2801 5.86011C7.1604 5.91503 7.05272 5.99306 6.96326 6.08971C6.87379 6.18636 6.8043 6.29973 6.75877 6.42331C6.71324 6.54689 6.69257 6.67824 6.69795 6.80983C6.70332 6.94142 6.73464 7.07066 6.7901 7.19011C8.2293 10.2729 10.7073 12.7509 13.7901 14.1901C14.0336 14.2901 14.3066 14.2901 14.5501 14.1901C14.6748 14.1455 14.7894 14.0766 14.8873 13.9873C14.9851 13.898 15.0643 13.7902 15.1201 13.6701L15.7401 12.2701C16.2571 12.455 16.7847 12.6086 17.3201 12.7301C17.5868 12.7901 17.8568 12.8401 18.1301 12.8801C18.3575 12.9289 18.5608 13.0554 18.705 13.238C18.8492 13.4205 18.9253 13.6476 18.9201 13.8801L18.9401 17.0001Z"
                fill="currentColor"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1v07gju"
          />
        </div>
        <div
          class="css-r2hvuq"
        >
          <div
            class="css-fnjvi5"
          >
            <svg
              class="chakra-icon css-16n3ul6"
              fill="none"
              focusable="false"
              height="16"
              viewBox="0 0 22 16"
              width="22"
            >
              <path
                d="M19 0H3C2.20435 0 1.44129 0.316071 0.87868 0.87868C0.31607 1.44129 0 2.20435 0 3V13C0 13.7956 0.31607 14.5587 0.87868 15.1213C1.44129 15.6839 2.20435 16 3 16H19C19.7956 16 20.5587 15.6839 21.1213 15.1213C21.6839 14.5587 22 13.7956 22 13V3C22 2.20435 21.6839 1.44129 21.1213 0.87868C20.5587 0.316071 19.7956 0 19 0ZM20 12.75L15.1 8.35L20 4.92V12.75ZM2 4.92L6.9 8.35L2 12.75V4.92ZM8.58 9.53L10.43 10.82C10.5974 10.9361 10.7963 10.9984 11 10.9984C11.2037 10.9984 11.4026 10.9361 11.57 10.82L13.42 9.53L18.42 14H3.6L8.58 9.53ZM3 2H19C19.1857 2.00149 19.3674 2.05467 19.5245 2.15358C19.6817 2.25249 19.8083 2.39322 19.89 2.56L11 8.78L2.11 2.56C2.19171 2.39322 2.31826 2.25249 2.47545 2.15358C2.63265 2.05467 2.81428 2.00149 3 2Z"
                fill="currentColor"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1v07gju"
          />
        </div>
      </div>
      <div
        class="chakra-stack css-1k9mbn1"
      >
        <div
          class="css-j7qwjs"
        >
          <h3
            class="chakra-text css-tbw3nr"
          >
            About:
          </h3>
          <p
            class="chakra-text css-xwguho"
          />
        </div>
        <div
          class="css-1i53ob6"
        >
          <h3
            class="chakra-text css-tbw3nr"
          >
            Education:
          </h3>
        </div>
        <div
          class="css-1xli2ds"
        >
          <h3
            class="chakra-text css-tbw3nr"
          >
            Expertise:
          </h3>
          <p
            class="chakra-text css-xwguho"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Profile Teacher should render value successfully 1`] = `
<div>
  <div
    class="chakra-stack css-8qeac1"
  >
    <div
      class="css-9npxv6"
    >
      <img
        alt="Background Student"
        class="chakra-image css-1m8o377"
        src="test-file-stub"
      />
      <div
        class="css-tyv5ff"
      >
        <div
          class="css-1367w0v"
        >
          <span
            class="chakra-avatar css-131besb"
          >
            <svg
              aria-label=" avatar"
              class="chakra-avatar__svg css-16ite8i"
              role="img"
              viewBox="0 0 128 128"
            >
              <path
                d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                fill="currentColor"
              />
              <path
                d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                fill="currentColor"
              />
            </svg>
          </span>
        </div>
        <h2
          class="chakra-text css-hn8sh4"
        >
          John Doe
        </h2>
        <p
          class="chakra-text css-7j6s3t"
        >
          Math
           Teacher
        </p>
      </div>
    </div>
    <div
      class="css-1j1ywq9"
    >
      <div
        class="chakra-stack css-10hx9m5"
      >
        <div
          class="css-r2hvuq"
        >
          <div
            class="css-fnjvi5"
          >
            <svg
              class="chakra-icon css-16n3ul6"
              fill="none"
              focusable="false"
              height="20"
              viewBox="0 0 16 20"
              width="16"
            >
              <path
                d="M8 0C5.87827 0 3.84344 0.842855 2.34315 2.34315C0.842855 3.84344 0 5.87827 0 8C0 13.4 7.05 19.5 7.35 19.76C7.53113 19.9149 7.76165 20.0001 8 20.0001C8.23835 20.0001 8.46887 19.9149 8.65 19.76C9 19.5 16 13.4 16 8C16 5.87827 15.1571 3.84344 13.6569 2.34315C12.1566 0.842855 10.1217 0 8 0ZM8 17.65C5.87 15.65 2 11.34 2 8C2 6.4087 2.63214 4.88258 3.75736 3.75736C4.88258 2.63214 6.4087 2 8 2C9.5913 2 11.1174 2.63214 12.2426 3.75736C13.3679 4.88258 14 6.4087 14 8C14 11.34 10.13 15.66 8 17.65ZM8 4C7.20887 4 6.43552 4.2346 5.77772 4.67412C5.11992 5.11365 4.60723 5.73836 4.30448 6.46927C4.00173 7.20017 3.92252 8.00444 4.07686 8.78036C4.2312 9.55628 4.61216 10.269 5.17157 10.8284C5.73098 11.3878 6.44371 11.7688 7.21964 11.9231C7.99556 12.0775 8.79983 11.9983 9.53073 11.6955C10.2616 11.3928 10.8864 10.8801 11.3259 10.2223C11.7654 9.56448 12 8.79113 12 8C12 6.93913 11.5786 5.92172 10.8284 5.17157C10.0783 4.42143 9.06087 4 8 4ZM8 10C7.60444 10 7.21776 9.8827 6.88886 9.66294C6.55996 9.44318 6.30362 9.13082 6.15224 8.76537C6.00087 8.39991 5.96126 7.99778 6.03843 7.60982C6.1156 7.22186 6.30608 6.86549 6.58579 6.58579C6.86549 6.30608 7.22186 6.1156 7.60982 6.03843C7.99778 5.96126 8.39991 6.00087 8.76537 6.15224C9.13082 6.30362 9.44318 6.55996 9.66294 6.88886C9.8827 7.21776 10 7.60444 10 8C10 8.53043 9.78929 9.03914 9.41421 9.41421C9.03914 9.78929 8.53043 10 8 10Z"
                fill="white"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1v07gju"
          >
            Jakarta
          </p>
        </div>
        <div
          class="css-r2hvuq"
        >
          <div
            class="css-fnjvi5"
          >
            <svg
              class="chakra-icon css-16n3ul6"
              fill="none"
              focusable="false"
              height="20"
              viewBox="0 0 21 20"
              width="21"
            >
              <path
                d="M18.4401 11.0001C18.2201 11.0001 17.9901 10.9301 17.7701 10.8801C17.3246 10.7819 16.8868 10.6516 16.4601 10.4901C15.9962 10.3213 15.4862 10.3301 15.0284 10.5147C14.5706 10.6993 14.1972 11.0467 13.9801 11.4901L13.7601 11.9401C12.7861 11.3983 11.8911 10.7253 11.1001 9.94011C10.3149 9.14913 9.64192 8.25411 9.1001 7.28011L9.5201 7.00011C9.96347 6.78303 10.3109 6.40964 10.4955 5.9518C10.6801 5.49396 10.6889 4.98402 10.5201 4.52011C10.3613 4.09253 10.231 3.65491 10.1301 3.21011C10.0801 2.99011 10.0401 2.76011 10.0101 2.53011C9.88867 1.82573 9.51973 1.18785 8.96972 0.731352C8.41972 0.274849 7.7248 0.0297211 7.0101 0.0401122H4.0101C3.57913 0.0360657 3.15235 0.124926 2.75881 0.300643C2.36527 0.47636 2.01421 0.734808 1.72953 1.05839C1.44485 1.38198 1.23324 1.76311 1.10909 2.17583C0.984942 2.58855 0.95118 3.02317 1.0101 3.45011C1.54284 7.63949 3.45613 11.532 6.44775 14.5127C9.43938 17.4935 13.3388 19.3926 17.5301 19.9101H17.9101C18.6475 19.9112 19.3595 19.6406 19.9101 19.1501C20.2265 18.8672 20.4792 18.5203 20.6516 18.1324C20.8239 17.7446 20.9121 17.3245 20.9101 16.9001V13.9001C20.8979 13.2055 20.6449 12.5367 20.1944 12.0078C19.744 11.4789 19.1239 11.1227 18.4401 11.0001ZM18.9401 17.0001C18.9399 17.1421 18.9095 17.2824 18.8509 17.4117C18.7923 17.5411 18.7068 17.6564 18.6001 17.7501C18.4884 17.8466 18.3577 17.9187 18.2165 17.9617C18.0753 18.0048 17.9267 18.0178 17.7801 18.0001C14.035 17.5199 10.5563 15.8066 7.89282 13.1304C5.2293 10.4542 3.53251 6.96745 3.0701 3.22011C3.05419 3.07363 3.06813 2.92544 3.1111 2.7845C3.15407 2.64356 3.22517 2.5128 3.3201 2.40011C3.41381 2.29344 3.52916 2.20795 3.65848 2.14933C3.7878 2.09071 3.92812 2.06029 4.0701 2.06011H7.0701C7.30265 2.05494 7.52972 2.13099 7.71224 2.27518C7.89476 2.41937 8.02131 2.62268 8.0701 2.85011C8.1101 3.12345 8.1601 3.39345 8.2201 3.66011C8.33562 4.18726 8.48936 4.70529 8.6801 5.21011L7.2801 5.86011C7.1604 5.91503 7.05272 5.99306 6.96326 6.08971C6.87379 6.18636 6.8043 6.29973 6.75877 6.42331C6.71324 6.54689 6.69257 6.67824 6.69795 6.80983C6.70332 6.94142 6.73464 7.07066 6.7901 7.19011C8.2293 10.2729 10.7073 12.7509 13.7901 14.1901C14.0336 14.2901 14.3066 14.2901 14.5501 14.1901C14.6748 14.1455 14.7894 14.0766 14.8873 13.9873C14.9851 13.898 15.0643 13.7902 15.1201 13.6701L15.7401 12.2701C16.2571 12.455 16.7847 12.6086 17.3201 12.7301C17.5868 12.7901 17.8568 12.8401 18.1301 12.8801C18.3575 12.9289 18.5608 13.0554 18.705 13.238C18.8492 13.4205 18.9253 13.6476 18.9201 13.8801L18.9401 17.0001Z"
                fill="currentColor"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1v07gju"
          >
            08123456789
          </p>
        </div>
        <div
          class="css-r2hvuq"
        >
          <div
            class="css-fnjvi5"
          >
            <svg
              class="chakra-icon css-16n3ul6"
              fill="none"
              focusable="false"
              height="16"
              viewBox="0 0 22 16"
              width="22"
            >
              <path
                d="M19 0H3C2.20435 0 1.44129 0.316071 0.87868 0.87868C0.31607 1.44129 0 2.20435 0 3V13C0 13.7956 0.31607 14.5587 0.87868 15.1213C1.44129 15.6839 2.20435 16 3 16H19C19.7956 16 20.5587 15.6839 21.1213 15.1213C21.6839 14.5587 22 13.7956 22 13V3C22 2.20435 21.6839 1.44129 21.1213 0.87868C20.5587 0.316071 19.7956 0 19 0ZM20 12.75L15.1 8.35L20 4.92V12.75ZM2 4.92L6.9 8.35L2 12.75V4.92ZM8.58 9.53L10.43 10.82C10.5974 10.9361 10.7963 10.9984 11 10.9984C11.2037 10.9984 11.4026 10.9361 11.57 10.82L13.42 9.53L18.42 14H3.6L8.58 9.53ZM3 2H19C19.1857 2.00149 19.3674 2.05467 19.5245 2.15358C19.6817 2.25249 19.8083 2.39322 19.89 2.56L11 8.78L2.11 2.56C2.19171 2.39322 2.31826 2.25249 2.47545 2.15358C2.63265 2.05467 2.81428 2.00149 3 2Z"
                fill="currentColor"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1v07gju"
          >
            john.doe@example.com
          </p>
        </div>
      </div>
      <div
        class="chakra-stack css-1k9mbn1"
      >
        <div
          class="css-j7qwjs"
        >
          <h3
            class="chakra-text css-tbw3nr"
          >
            About:
          </h3>
          <p
            class="chakra-text css-xwguho"
          />
        </div>
        <div
          class="css-1i53ob6"
        >
          <h3
            class="chakra-text css-tbw3nr"
          >
            Education:
          </h3>
          <ul
            class="css-ydu6ez"
            role="list"
          >
            <li
              class="css-1vcosx7"
            >
              Bachelor of Science
              , 
              University of Example
            </li>
            <span
              class="chakra-text css-cif1vy"
            >
              2015
               - 
              2020
            </span>
          </ul>
        </div>
        <div
          class="css-1xli2ds"
        >
          <h3
            class="chakra-text css-tbw3nr"
          >
            Expertise:
          </h3>
          <p
            class="chakra-text css-xwguho"
          >
            Mathematics
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
